### PR TITLE
PHPC-2101: Default Binary constructor  parameter to TYPE_GENERIC

### PR DESCRIPTION
**Pull request by levon80999**
https://jira.mongodb.org/browse/PHPC-2101

### CODE REVIEW

File: tests/bson/bson-binary-002.phpt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+--TEST--
+MongoDB\BSON\Binary::__construct() defaults to TYPE_GENERIC
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
```
Comment (jmikola) [inline tests/bson/bson-binary-002.phpt:6]: This include is not necessary and can be removed. It was only included in `bson-binary-001.phpt` for some helper functions used within the test (e.g. `fromPHP()`).

**Review by jmikola — CHANGES_REQUESTED** [review]
Please check the CI failures for this PR. There looks to be another test that requires changes.

File: tests/bson/bson-binary-002.phpt (hunk 1 above)
Comment (levon80999) [reply tests/bson/bson-binary-002.phpt]: Done

**Comment by levon80999** [comment]
Seems I accidentally closed this PR when tried to solve conflict in phpstorm editor. new PR link is this https://github.com/mongodb/mongo-php-driver/pull/1362

**Comment by jmikola** [comment]
> Seems I accidentally closed this PR when tried to solve conflict in phpstorm editor.

In the future, you should just be able to re-open the pull request. If you don't have perms to do so, please reach out to @tom-selander.

**Comment by alcaeus** [comment]
@jmikola that's usually possible, but when force-pushing to a branch that's part of a closed pull request GotHub prevents one from re-opening said PR. I suspect that's what happened here.

**Pull request closed without merging**